Harvest.Web/ClientApp/src/Quotes/QuoteContainer.tsx

```diff
@@ -285,22 +286,34 @@ export const QuoteContainer = () => {
               </ul>
             </div>
             <div className="row justify-content-center">
-              <button
-                className="btn btn-link mt-4"
-                onClick={() => save(false)}
-                disabled={notification.pending || formErrorCount > 0}
-              >
-                Save Quote
-              </button>
-              <button
-                className="btn btn-primary mt-4"
-                onClick={() => save(true)}
-                disabled={
-                  notification.pending || !isValid() || formErrorCount > 0
-                }
-              >
-                Submit Quote
-              </button>
+              {/* Supervisor can only see save option, so it becomes a primary button.  FM can do both save and submit */}
+              <ShowFor roles={["Supervisor"]}>
+                <button
+                  className="btn btn-primary mt-4"
+                  onClick={() => save(false)}
+                  disabled={notification.pending || formErrorCount > 0}
+                >
+                  Save Quote
+                </button>
+              </ShowFor>
+              <ShowFor roles={["FieldManager"]}>
```

Comment:
Didn't test, but I think you need bother roles here?

Reply:
I don't think so, because the prior `<ShowFor/>` handles `Supervisor` role.

Reply:
yea, I wanted to get fancy here and have different buttons be styled different depending on the role.  This seemed the cleanest way to do it.